### PR TITLE
Load generator env terraform

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

We want to have an environment separate from the wargames env that provides load so that the traffic isn't coming from within so to speak.
# What has changed

Terraform files and apply script and associated files added.
# How to test?

Look at the rm loadgen env on GCP and check it out, alternatively build your own by following the instructions in the readme within the loadgen-terraform folder.
# Links

https://trello.com/c/ChYRVvPr/1193-environment-to-provide-load-generation-for-wargaming-capacity-testing-8

### CODE REVIEW

**Comment by hbrace** [comment]
[Environment to provide load generation for wargaming/capacity testing (8?)](https://trello.com/c/ChYRVvPr/1193-environment-to-provide-load-generation-for-wargaming-capacity-testing-8)

File: loadgen-terraform/env_config.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RABBIT_PASS=
```
Comment (NickGrantX) [inline loadgen-terraform/env_config.txt:1]: All blanks? Why?

**Review by NickGrantX — APPROVED** [review]
Eyeballed it. Questions but happy to approve

File: loadgen-terraform/env_config.txt (hunk 1 above)
Comment (hbrace) [reply loadgen-terraform/env_config.txt:1]: Didn't want to put ips and passwords in a public repo, I know they're only for the wargames env but I felt it was safer to not.

**Review by AdamHawtin — APPROVED** [review]
LGTM

**Pull request merged**